locale/sv/LC_MESSAGES/django.po

```diff
@@ -536,11 +536,18 @@ msgstr "Felaktig pk \"{pk_value}\" - objektet existerar inte."
 msgid "Invalid value in filter %(filter)s"
 msgstr "Felaktigt värde i filter %(filter)s"
 
+msgid "You need to login with PIKI library card to reserve this resource."
+msgstr ""
```

Comment:
"Du måste logga in med en PIKI resursenbibliotekskort för att reservera den här resursen."